connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -47,6 +47,13 @@ message Relation {
 
     Unknown unknown = 999;
   }
+  // Optional. Every relation might have an alias.
+  Alias alias = 200;
+
+  // Relation alias.
+  message Alias {
```

Comment:
why can't we use string directly?

Reply:
do you think if we need to care client  call `xx.as("")`? Does client side should reject this? If so we can just use a string.

Reply:
oh, I thought the default value of string is null...

Reply:
default is empty string...

I really don't like the way of proto for this..